scripts/reset-lang-cache.js

```diff
@@ -0,0 +1,16 @@
+module.exports = () => {
+  const links = document.querySelectorAll('[href^="/languages/"]')
+  for (const link of links) {
+    link.addEventListener('click', (e) => {
+      e.preventDefault()
+
+      window.caches.keys()
```

Comment:
Do we need to do any feature detection around this block?

Reply:
Updated